### PR TITLE
Beaver triple gen

### PR DESCRIPTION
**Pull request by gmuraru**
Fix for #2708

What changed:
* Flatten a, b, c before concatenating - this was needed since we can not simply stack the tensors if they have different dimensions

### CODE REVIEW

File: syft/frameworks/torch/crypto/beaver.py [hunk 1]
```diff
@@ -27,7 +27,13 @@ def request_triple(
     a = torch.randint(field, a_size)
     b = torch.randint(field, b_size)
     c = cmd(a, b)
-    a_shared = a.share(*locations, field=field, crypto_provider=crypto_provider).child
-    b_shared = b.share(*locations, field=field, crypto_provider=crypto_provider).child
-    c_shared = c.share(*locations, field=field, crypto_provider=crypto_provider).child
+
+    res = torch.cat((a.view(-1), b.view(-1), c.view(-1)))
```
Comment (murarugeorgec) [inline syft/frameworks/torch/crypto/beaver.py]: @LaRiffle those should be moved to the cryptoProvider?

File: syft/frameworks/torch/crypto/beaver.py [hunk 2]
```diff
@@ -27,7 +27,13 @@ def request_triple(
...
+
+    shares = res.share(*locations, field=field, crypto_provider=crypto_provider).child
+
+    a_shared = shares[: a.numel()].reshape(a_size)
+    b_shared = shares[a.numel() : -c.numel()].reshape(b_size)
+    c_shared = shares[-c.numel() :].reshape(c.shape)
+
```
Comment (midokura-silvia) [inline syft/frameworks/torch/crypto/beaver.py]: Is there a unit test testing this code?

Comment (gmuraru) [reply syft/frameworks/torch/crypto/beaver.py]: There are already tests that are testing the ```request_triple``` method - should I add more?

Comment (midokura-silvia) [reply syft/frameworks/torch/crypto/beaver.py]: if it is already tested, it's fine, thanks.

**Comment by gmuraru** [comment]
I will continue working on the first item from @LaRiffle options and then come back to this issue.

**Comment by iamtrask** [comment]
> I will continue working on the first item from @LaRiffle options and then come back to this issue.

Looks like this is unblocked - great work on https://github.com/OpenMined/PySyft/pull/2741!

File: syft/exceptions.py [hunk 3]
```diff
@@ -273,11 +273,11 @@ def __init__(self, obj_id, worker):
             + " which does not exist!!! "
         )
         message += (
-            "Use .send() and .get() on all your tensors to make sure they're"
+            "Use .send() and .get() on all your tensors to make sure they're "
```
Comment (gmuraru) [inline syft/exceptions.py]: Not related to the issue. When a string is split on multiple lines like this the printed resulted string would be all the chunks concatenated - need to add space between words :)

**Pull request merged**